roles/bootstrap-os/tasks/bootstrap-centos.yml

```diff
@@ -89,6 +92,9 @@
 # libselinux-python is required on SELinux enabled hosts
 # See https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#managed-node-requirements
 - name: Install libselinux python package
+  vars:
+    ansible_python_interpreter: "{{ (ansible_os_family == 'RedHat' and ansible_distribution_version < '8') | \
+ternary('/usr/bin/python', temp_ansible_python_interpreter ) }}"
```

Comment:
The molecule_tests job is failed here like
```
TASK [bootstrap-os : Install libselinux python package] ************************
fatal: [centos7]: FAILED! => {"msg": "'ansible_os_family' is undefined"}
```
Ref: https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/1273049784#L370